landoscript/tests/test_android_l10n_import.py

```diff
@@ -242,7 +244,6 @@ async def test_success(
         "android_l10n_import_info": android_l10n_import_info,
     }
     # done here because setup_test sets up github_installation_response too soon...argh
-    from yarl import URL
```

Comment:
Was the comment meant to go with the removed line?  (either way, seems unrelated with this patch?)

Reply:
The comment is about repeating all of the test set up overall (instead of using `setup_test`), but I can see how it is...not particularly great. I'll improve it.

The import change is unrelated, yeah. It's not harmful, but I'll revert it to keep things clean. (Another follow up I'm planning is to add isort, which will help clean some of these things up.)